### DISCUSSION TITLE
About Verify

### DISCUSSION THREAD

**Discussion by heebaebang** [WalletConnect/walletconnect-monorepo#3588] [Developer Support] [answered]
I received an email about [Verify](https://preview.mailerlite.io/emails/webview/164487/97945158983616379).

My question is how can I expose the verified mark(Verified domain)?
![image](https://github.com/WalletConnect/walletconnect-monorepo/assets/121843393/62bce646-5b25-42ae-9e02-8f573e4c287c)

Is this handled by a wallet connection platform like Rainbow or Metamask?

If I have verified my domain, where can I see the verified domain?
We are servicing a dapp.

**Comment by glitch-txs** [comment, marked as answer]
In order to verify your dapp you can follow [this steps](https://docs.walletconnect.com/2.0/cloud/verify)

For more info you can take a look at the specs [here](https://specs.walletconnect.com/2.0/specs/clients/core/verify/verify-attestations) :)
